NewSpring/ApollosProject
# [DEV-3764] Adds NS Branded Logo and Updates Landing Screen

## DESCRIPTION

We currently don't have the NS Branded Logo in the app and the Apollos Logo is shown on load.

### What does this PR do, or why is it needed?

![Screen Shot 2019-05-13 at 11 31 07 AM](https://user-images.githubusercontent.com/8878532/57634308-12e1fd80-7573-11e9-9463-770cc3183389.png)

This PR adds the NS branding icons in the app and updates accordingly.
The icons are : `LeafOutline`, `LeafSolid`, `LeafLined`, and `LeafLogo`

### How do I test this PR?

Make sure you're signed out and launch onboarding. Make sure that the first landing screen has our logo.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] Closes [DEV-3764](https://newspring.atlassian.net/secure/RapidBoard.jspa?rapidView=71&modal=detail&selectedIssue=DEV-3764)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
